pkg/ttnpb/udp/data_rate.go

```diff
@@ -67,8 +67,8 @@ func (d *DataRate) UnmarshalJSON(data []byte) error {
 
 var (
 	errDataRate = errors.DefineInvalidArgument("data_rate", "invalid data rate")
-	sfRegexp    = regexp.MustCompile("^SF(6|7|8|9|10|11|12)")
-	drRegexp    = regexp.MustCompile("BW(125|250|500)$")
+	sfRegexp    = regexp.MustCompile("^SF(5|6|7|8|9|10|11|12)")
+	drRegexp    = regexp.MustCompile("BW(\\d{2,3})$")
```

Comment:
Shouldn't we just make this a `\d+`? Perhaps even add a `(?:\.\d+)?`. The gateway module also supports `62.5`, `31.25`, `15.625`, `7.8125`, so I can imagine those may be used in the future too.